src/connection_string.ts

```diff
@@ -572,6 +573,15 @@ export const OPTIONS = {
   autoEncryption: {
     type: 'record'
   },
+  serverApi: {
```

Comment:
Does this mean we are allowing the user to specify the api version in the connection string? The spec specifically states not to allow it as we [don't want developers changing behaviour without code changes](https://github.com/mongodb/specifications/blob/master/source/versioned-api/versioned-api.rst#no-uri-options).

Reply:
Ok, I added a check to throw an error if it is provided in the URI options, and a corresponding test.